lecli/team_api.py

```diff
@@ -0,0 +1,185 @@
+    except requests.exceptions.RequestException as error:
+        print error
+        exit(1)
+
+
+def get_team(team_id):
+    """
+    Get a specific team.
+    """
+    headers = apiutils.generate_headers('rw')
+    params = {'teamid': team_id}
+    try:
+        response = requests.request('GET', _url() + "/" + team_id, params=params,
+                                    headers=headers)
+        handle_get_teams_response(response)
+    except requests.exceptions.RequestException as error:
+        print error
+        exit(1)
+
+
+def create_team(name):
+    """
+    Add a new user to the current account.
+    """
+    params = {
+        'team': {
+            'name': str(name),
+            'users': []
+        }
+    }
+    headers = apiutils.generate_headers('rw')
+
+    try:
+        response = requests.post(_url(), json=params, headers=headers)
+        if response.status_code == 201:
+            print 'Team created with name: %s' % name
+    except requests.exceptions.RequestException as error:
+        print error
+        exit(1)
+
+
+def delete_team(team_id):
+    url = _url() + '/' + team_id
+    headers = apiutils.generate_headers('rw')
+
+    try:
+        response = requests.delete(url, headers=headers)
+        if response_error(response) is True:  # Check response has no errors
+            print 'Delete team failed, status code: ' + str(response.status_code)
+            exit(1)
+        elif response.status_code == 204:
+            print 'Deleted team with id: %s' % team_id
+    except requests.exceptions.RequestException as error:
+        print error
+        exit(1)
+
+
+def rename_team(team_id, team_name):
+    url = _url() + '/' + team_id
+    params = {
+        'team': {
+            'name': team_name,
+            'users': [
+                {'id': ''}
+            ]
+        }
+    }
+    headers = apiutils.generate_headers('rw')
+
+    try:
+        response = requests.patch(url, json=params, headers=headers)
+        if response_error(response) is True:  # Check response has no errors
+            print 'Renaming team with id: %s failed, status code: %s' \
+                  % (team_id, str(response.status_code))
+            exit(1)
+        elif response.status_code == 200:
+            print "Team: '%s' renamed to: '%s'" % (team_id, team_name)
+    except requests.exceptions.RequestException as error:
+        print error
+        exit(1)
+
+
+def add_user_to_team(team_id, user_id):
+    headers = apiutils.generate_headers('rw')
+    params = {'teamid': team_id}
+    try:
+        response = requests.request('GET', _url() + '/' + team_id, params=params,
+                                    headers=headers)
+        if response.status_code == 200:
+            url = _url() + '/' + team_id
+            params = {
+                'team': {
+                    'name': response.json()['team']['name'],
+                    'users': [
+                        {'id': user_id}
```

Comment:
This will remove the current team members and only the new user will be added, in the JSON we need to reference the users field from the above response and append `{'id': user_id}` to it